jobs/buildhub/utils.py

```diff
@@ -150,6 +155,22 @@ def build_record_id(record):
     return id_.replace('.', '-').lower()
 
 
+# Compile these regexes once per module to speed up their use inside
+# the is_build_url() function.
+
+_build_url_exclude_names_regex = re.compile(
+    '.+(tinderbox|try-builds|partner-repacks|latest|contrib|/0\.|'
+    'experimental|namoroka|debug|sha1-installers|candidates/archived|'
+    'stylo-bindings|/1.0rc/|/releases/win../|dominspector|/test/|testing|'
+    '%28.+%29|\sInstaller\.(\w{2,3}\-?\w{0,3})\.exe)'
+)
+
+_build_url_exclude_suffixes_regex = re.compile(
+    '.+(sdk|tests|crashreporter|stub|gtk2.+xft|source|asan)'
+)
+
```

Comment:
I just moved these outside the function since they're the same every time. This way the `re.compile` can be called *once* for the whole scraper run instead of thousands of times.